test/jdk/security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java

```diff
@@ -433,6 +476,7 @@
  * @library /test/lib
  * @build jtreg.SkippedException ValidatePathWithURL CAInterop
  * @run main/othervm -Djava.security.debug=certpath,ocsp CAInterop emsignrootcag1 OCSP
```

Comment:
eMudhra OCSP responders don't support GET calls so these tests fail, refer - [JDK-8328830](https://bugs.openjdk.org/browse/JDK-8328830). We are following up with CA. Meanwhile, can you please add these test to ProblemList at https://github.com/openjdk/jdk/blob/master/test/jdk/ProblemList.txt#L627

```
# jdk_security_infra

security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java#emsigneccrootcag3 8328830 generic-all 
security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java#emsignrootcag1    8328830 generic-all
```

Reply:
Oooooh! So we found the CA that does not like GET requests, nice. I think these tests should be problemlisted with [JDK-8328830](https://bugs.openjdk.org/browse/JDK-8328830) subtask, and the fix for [JDK-8328830](https://bugs.openjdk.org/browse/JDK-8328830) would then fix/resolve the issue?

Reply:
Sure, I will do that then. Thanks!